XenAdmin/Wizards/ImportWizard/ImportSelectNetworkPage.cs

```diff
@@ -51,6 +53,32 @@ protected override bool ImplementsIsDirty()
             return true;
         }
 
+        public override bool ShowNetwork(Host targetHost, XenAPI.Network network)

```

Comment:
Why are you repeating the base class logic, instead of adding only the Sriov bit and then call return base.ShowNetwork?